x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -19,83 +19,93 @@
       }
     }
   , { "set":
-      { "field": "ecs.event.type"
+      { "field": "event.type"
       , "value": "{{suricata.eve.event_type}}"
       }
     }
 
   , { "set":
-      { "field": "ecs.user_agent.raw"
+      { "field": "user_agent.original"
       , "value": "{{suricata.eve.http.http_user_agent}}"
       }
     }
   , { "user_agent":
-      { "field": "ecs.user_agent.raw"
-      , "target_field": "ecs.user_agent"
+      { "field": "user_agent.original"
+      , "target_field": "user_agent"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os"
-      , "target_field": "ecs.user_agent.temp_os"
+      { "field": "user_agent.os"
+      , "target_field": "user_agent.temp_os"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.temp_os"
-      , "target_field": "ecs.user_agent.os.full_name"
+      { "field": "user_agent.temp_os"
+      , "target_field": "user_agent.os.full_name"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_name"
-      , "target_field": "ecs.user_agent.os.name"
+      { "field": "user_agent.os_name"
+      , "target_field": "user_agent.os.name"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_version"
-      , "target_field": "ecs.user_agent.os.version"
+      { "field": "user_agent.os_version"
+      , "target_field": "user_agent.os.version"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_major"
-      , "target_field": "ecs.user_agent.os.major"
+      { "field": "user_agent.os_major"
+      , "target_field": "user_agent.os.major"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_minor"
-      , "target_field": "ecs.user_agent.os.minor"
+      { "field": "user_agent.os_minor"
+      , "target_field": "user_agent.os.minor"
       , "ignore_missing": true
       }
     }
 
   , { "geoip":
       { "field": "suricata.eve.src_ip"
```

Comment:
It looks like we should "promote" this up to the `source.ip`.

Reply:
Same answer as above, I was refraining from using `source`.